src/main/resources/assets/opencomputers/doc/de_DE/block/charger.md

```diff
@@ -0,0 +1,9 @@
+# Ladegerät
+
+![Gehen wir es an!](oredict:oc:charger)
+
+Das Ladegerät wird verwendet, um Geräte wie den [Roboter](robot.md), [Drohnen](../item/drone.md) und [Tablets](../item/tablet.md) zu laden. Ein Ladegerät muss mit einem Redstonesignal aktiviert werden. Die Ladegeschwindigkeit hängt von der Stärke des Redstonesignals ab. Eine Stärke von 15 bedeutet 100% Ladegeschwindigkeit.
+
+Es gilt zu bedenken, dass umgestellt werden kann, wenn das Ladegerät mit einem [Schraubenschlüssel](../item/wrench.md) berührt wird. In diesem Modus ist die Standardgeschwindigkeit 100% welche mit einem Redstonesignal verringert werden kann.
+
+Wenn ein Tablet in einem Ladegerät ist wird die erste Festplatte mit angeschlossenen [Computern](../general/computer.md) verbunden, ähnlich wie es bei [Disketten](../item/floppy.md) in [Diskettenlaufwerken](diskDrive.md) der Fall ist. So könne Daten zwischen dem Computer und dem Tablet ausgetauscht werden.
```

Comment:
* `Wenn ein Tablet in einem Ladegerät ist` -> `Wenn ein Tablet sich in einem Ladegerät befindet`
* `So könne Daten` -> `So können Daten`